.github/workflows/test.yaml

```diff
@@ -0,0 +1,24 @@
+name: Test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+permissions: {}
+
+jobs:
+  go-test:
+    name: Run Go tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: '1.17'
```

Comment:
Why Go 1.17?

Reply:
I'm not particularly attached to any version, I went for 1.17 as that seems to be what we use for building Teleport at the moment. I'm happy to switch to basically any value here.